.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: go
 go:
-  - 1.x
```

Comment:
the smallest of bikesheds, my current mindset is to prefer `1.x` for libraries that have relatively infrequent updates so every update doesn't have to fiddle with this value.

Reply:
no problem, I can put that back